### PR TITLE
Update user accounts on rewrite if updated on legacy (fixes #329)

### LINKED ISSUE #329: User names are not updated from Legacy
Alina changed the name of one of the OSC textbook authors. The name did not update on Rewrite, but it did on Legacy. We need to add a trigger that updates users on Rewrite when a user is updated on Legacy.

### PR DESCRIPTION
**Pull request by mwharrison**

### CODE REVIEW

File: cnxarchive/tests/test_database.py [hunk 1]
```diff
@@ -1539,6 +1539,7 @@ def test_new_module_wo_uuid(self, cursor):
         self.assertEqual(uuid_, controls_uuid)
         self.assertEqual(license_id, controls_license_id)
 
+
```
Comment (karenc) [inline cnxarchive/tests/test_database.py]: Could you remove the extra blank lines around methods?
There should only be one blank line between methods.

Running pep8 gives:
```
cnxarchive/tests/test_database.py:1543:5: E303 too many blank lines (2)
cnxarchive/tests/test_database.py:1602:5: E303 too many blank lines (2)
cnxarchive/tests/test_database.py:1642:5: E303 too many blank lines (2)
```

I know our code is not yet pep8 compliant, but we're trying to be. :)

Comment (mwharrison) [inline cnxarchive/tests/test_database.py]: Done!
I was trying to use the Atom editor, seems it like to take a little more liberty with changing things than I would like - sorry about that!

**Comment by karenc** [comment]
:+1:

**Comment by pumazi** [comment]
:+1:

**Pull request merged**